scripts/west_commands/runners/pyocd.py

```diff
@@ -161,13 +169,23 @@ def flash(self, **kwargs):
             fformat_args = ['--format', 'elf']
         else:
             fname = self.elf_name
-        if not os.path.isfile(fname):
+        if fname is None:
+            raise ValueError(
+                'No {} file provided. '.format(self.flash_format))
+        elif not os.path.isfile(fname):
             raise ValueError(
                 'Cannot flash; ({}) file not found. '.format(fname))
 
+        ''' If running in WSL, the path needs to be a windows path '''
+        ''' Convert to relative path first so the drive mount point '''
+        ''' does not need to be converted '''
+        if self.wsl_py_path is not None: 
+            fname = os.path.relpath(fname)
+            fname = fname.replace('/', '\\')
```

Comment:
Is this a job for pathlib.Path.as_posix() instead?

Reply:
In this case we are changing from posix to nt.  Inside WSL the paths are posix, but we need to convert them to Windows paths for use with 'py.exe -m pyocd'